src/exercism/Darts.java

```diff
@@ -0,0 +1,18 @@
+class Darts {
+    static int score(double xOfDart, double yOfDart) {
+        double userRadius = Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2);
```

Comment:
Please replace Math.pow(x, 2) with "x * x" everywhere. No need to use power for simple multiplication